Cannot apply hunk 0 for file node_modules/lottie-react-native/src/android/build.gradle
```diff
@@ -3,11 +3,12 @@ def safeExtGet(prop, fallback) {
 }
 
 apply plugin: 'com.android.library'
+apply from: 'gradle-maven-push.gradle'
 
 //def DEFAULT_ANDROID_SUPPORT_LIB_VERSION = "1.0.0"
 
 android {
-  compileSdkVersion safeExtGet('compileSdkVersion', 30)
+  compileSdkVersion 36
 
   defaultConfig {
     minSdkVersion safeExtGet('minSdkVersion', 21)
```
